phpBB/phpbb/template/twig/extension.php

```diff
@@ -71,7 +71,7 @@ public function getFilters()
 	{
 		return array(
 			new \Twig_SimpleFilter('subset', array($this, 'loop_subset'), array('needs_environment' => true)),
-			new \Twig_SimpleFilter('addslashes', 'addslashes'),
+			new \Twig_SimpleFilter('addslashes', 'addslashes'), // Deprecated 3.2.0
```

Comment:
I'd prefer if you'd add the comment above this. I think it should say this:
```@deprecated 3.2.0 Uses twig's JS escape method instead of addslashes```